src/common/options/ceph-exporter.yaml.in

```diff
@@ -0,0 +1,54 @@
+# -*- mode: YAML -*-
+---
+
+options:
+- name: exporter_sock_dir
+  type: str
+  level: advanced
+  desc: The path to ceph daemons socket files dir
+  default: /var/run/ceph/
+  services:
+  - ceph-exporter
+  flags:
+  - runtime
+- name: exporter_addr
+  type: str
+  level: advanced
+  desc: Host ip address where exporter is deployed
+  default: 0.0.0.0
+  services:
+  - ceph-exporter
+- name: exporter_port
+  type: int
+  level: advanced
+  desc: Port to deploy exporter on. Default is 9926
```

Comment:
How did you choose port 9926 over any other port?

What will the TLS port be?

(I'd like to know so we could add this to firewalld's definitions.)

There are a lot of definitions in the code for `exporter_port`. Maybe you should rename these to `exporter_http_port` to distinguish from a future `exporter_https_port`

Reply:
> How did you choose port 9926 over any other port?

I see now, you registered that `9926` free port at https://github.com/prometheus/prometheus/wiki/Default-port-allocations

Reply:
@ktdreyer  TLS isn't supported yet for the exporter server,so for now its just http port I guess.

Reply:
Right, I'm suggesting we rename *now* to avoid confusion

Reply:
Done!